src/atoms/Button/Button.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { StyledButton, ButtonProps } from './Button';
+
+import { SearchButtonText } from 'context';
+
+export default {
+  title: 'TMS/Buttons',
+  component: StyledButton,
+} as Meta;
+
+const Template: Story<ButtonProps> = (args) => <StyledButton {...args}/>;
+
+export const SearchButton = Template.bind({});
+
+SearchButton.args = {
+  text: SearchButtonText,
+  onClick: action('search'),
+}
+
+export const TestButton = Template.bind({});
+
+TestButton.args = {
+  ...SearchButton.args,
+  text: 'Test Button',
+  btnColor: '#000000'
```

Comment:
Move all colors to some file and name them, ex:

export const Colors = {
        black:  '#000000',
        white: '#FFF'
}

Reply:
btnColor is also a color :)